### ISSUE TITLE
satisfies keyword causes fatal error

### ISSUE THREAD

**Issue by JacobWeisenburger** [denoland/fresh#984]
Is there a way to use the `satisfies` keyword? because when I try to do it, it causes a fatal error

```ts
const defaults = {
    type: 'page',
    robots: { index: true, follow: true },
} satisfies {
    type: Types
    robots: Robots
}
```

```ts
X [ERROR] Expected ";" but found "satisfies"
      15 │ } satisfies {
```

**Comment by lino-levan** [comment]
What deno version are you running (`deno --version`)?

**Comment by JacobWeisenburger** [comment]
```
$ deno --version
deno 1.29.1 (release, x86_64-pc-windows-msvc)
v8 10.9.194.5
typescript 4.9.4
```

**Comment by lino-levan** [comment]
Just looked into this. Looks like the reason this works in routes but fails on islands is because fresh's version of esbuild is out of date. 
- Esbuild added support for satisfies in [v0.15.13](https://github.com/evanw/esbuild/releases/tag/v0.15.13)
- Fresh currently uses v0.14.51

I'm opening a PR to bump the dependency. Thanks for reporting @JacobWeisenburger!

**Comment by JacobWeisenburger** [comment]
great, thanks

**Comment by JacobWeisenburger** [comment]
When do you think this will be in a release?

**Comment by lino-levan** [comment]
When https://github.com/lucacasonato/esbuild_deno_loader/pull/22 lands but I don't know if anyone other than @lucacasonato can merge that PR.

**Comment by lino-levan** [comment]
For curious observers who have not been following this super stringently, upstream landed the update that we needed and https://github.com/denoland/fresh/pull/986 is now in the green so hopefully that gets reviewed soon!

**Comment by JacobWeisenburger** [comment]
Any news on this? Cuz I'm still getting the error.

```
X [ERROR] Expected ";" but found "satisfies"
      15 │ } satisfies {
```

```
deno 1.32.0 (release, x86_64-pc-windows-msvc)
v8 11.2.214.9
typescript 5.0.2
```

**Comment by lino-levan** [comment]
This should be fixed in a new tagged release. @kt3k when do you think the next tag will happen?

**Comment by kt3k** [comment]
Probably in the next week. cc @hashrock

**Comment by JacobWeisenburger** [comment]
Any news on this?

**Comment by kt3k** [comment]
Sorry for the delay. We released v1.1.5 https://github.com/denoland/fresh/releases/tag/1.1.5

BTW the latest Deno CLI is not compatible with Fresh temporarily (which will be fixed in the next patch release). Please use Deno 1.32.1 or below versions to use fresh for now.
